### DISCUSSION TITLE
Problema con mi motherboard

### DISCUSSION THREAD

**Discussion by Itcheb** [community/community#62603] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hola a todos, tengo un problema con mi motherboard mks robin nano v1 3, el hotend esta conectado a HE0 conector, pero este me dejo de funcionar, la temperatura sube constantemente sin parar, pude resolverlo conectandolo a HE1 y ahi funciona bien, el tema es que solo puedo precalentar el extursor en Cura usando precalentar extursor 2, me gustaria saber que deberia modificar  o como modificar el firmware para anular el HE0 y que solo este activado HE1 asi poder precalentar directamente desde la pantalla de la impresora (Hellbot Magna 2 230) y tambien que Cura comieze a precalentar HE1 cuando envio una impresion, desde ya muchas gracias, saludos

**Comment by ManishParashar0** [comment]
Para solucionar el problema de tu motherboard MKS Robin Nano v1.3 y configurarla para que solo funcione con el conector HE1 y puedas precalentar el extrusor directamente desde la pantalla de la impresora y desde Cura, necesitarás modificar el firmware de la impresora. A continuación, te doy los pasos generales para hacerlo, pero ten en cuenta que modificar el firmware puede tener riesgos, por lo que es importante hacer una copia de seguridad y asegurarte de tener el conocimiento adecuado antes de continuar.

1. **Obtener el firmware**: Primero, asegúrate de tener una copia del firmware actual de tu impresora. Si no lo tienes, puedes buscarlo en el sitio web del fabricante o en el repositorio oficial del firmware de tu impresora en GitHub u otra plataforma.

2. **Instalar el entorno de desarrollo**: Necesitarás instalar el entorno de desarrollo de Arduino para poder compilar y cargar el firmware en la placa MKS Robin Nano. Puedes obtener Arduino IDE desde su sitio web oficial.

3. **Modificar el firmware**: Abre el firmware en el Arduino IDE y busca el archivo de configuración correspondiente al pinout de la placa MKS Robin Nano. Es posible que el archivo se llame "pins.h" o algo similar. Busca las líneas donde se definen los pines HE0 y HE1.

4. **Desactivar HE0**: Deberías poder desactivar el pin HE0 comentando o eliminando la línea que lo define. También, asegúrate de que todas las referencias a HE0 en el firmware estén desactivadas o eliminadas.

5. **Configurar HE1**: Asegúrate de que el pin HE1 esté correctamente configurado y activado en el firmware.

6. **Guardar y compilar**: Una vez realizadas las modificaciones, guarda los cambios y compila el firmware para asegurarte de que no haya errores.

7. **Cargar el firmware**: Conecta la placa MKS Robin Nano a tu computadora mediante un cable USB y carga el firmware compilado en la placa utilizando el Arduino IDE.

8. **Prueba**: Después de cargar el firmware modificado, prueba la impresora para asegurarte de que solo esté activo el conector HE1 y de que puedes precalentar el extrusor desde la pantalla de la impresora y desde Cura.

Ten en cuenta que los pasos específicos y los nombres de los archivos pueden variar según el firmware y la versión de la placa que estés utilizando. Es importante seguir las instrucciones proporcionadas por el fabricante y realizar cambios con precaución para evitar dañar la placa o perder la garantía.

Si no te sientes cómodo modificando el firmware o encuentras algún problema, es recomendable buscar ayuda en comunidades de impresión 3D o foros especializados donde otros usuarios con experiencia puedan brindarte asistencia específica para tu impresora y placa.

**Comment by Itcheb** [comment]
Muchas Gracias!!!, entendi perfecto, voy a ver que puedo hacer, gracias, saludos
